testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/ClientPoliciesTest.java

```diff
@@ -87,6 +87,7 @@
 import org.keycloak.testsuite.admin.ApiUtil;
 import org.keycloak.testsuite.arquillian.annotation.AuthServerContainerExclude;
 import org.keycloak.testsuite.arquillian.annotation.AuthServerContainerExclude.AuthServer;
+import org.keycloak.testsuite.arquillian.annotation.EnableFeature;
```

Comment:
It seems that it is not used.

Reply:
Thanks, I've removed this and also some more unused imports from ClientPoliciesTest class. Could you please re-check?

Reply:
@mposolda I've checked them. Thank you.